src/promptflow-devkit/promptflow/_sdk/entities/_chat_group/_chat_role.py

```diff
@@ -28,10 +30,46 @@ class ChatRole:
     :type inputs: Optional[Dict]
     """
 
-    def __init__(self, flow: Union[str, PathLike], role: str, inputs: Optional[Dict] = None, **kwargs):
+    def __init__(self,
+                 flow: Union[str, PathLike],
+                 role: str,
+                 inputs: Optional[Dict] = None,
+                 name: Optional[str] = None,
+                 stop_signal: Optional[str] = None,
+                 working_dir: Optional[Path] = None,
+                 connections: Optional[Dict[str, Any]] = None,
+                 inputs_mapping: Optional[Dict[str, str]] = None,
+                 executor_client: Optional[Any] = None,
+                 environment_variables: Optional[dict] = None,
+                 log_path: Optional[Path] = None,
+                 output_dir: Optional[Path] = None,
+                 worker_count: Optional[int] = None,
+                 line_timeout_sec: Optional[int] = None,
+                 init_kwargs: Optional[Dict[str, Any]] = None,
+                 **kwargs):
         self._role = role
         self._flow, self._flow_object = self._validate_flow(flow)
```

Comment:
Here the expected input of flow parameter should be something like "flow.dag.yaml", not the complete path.

Reply:
Fixed. This issue wasn't caught in e2e test since it doesn't call executor server. I fixed the path issue and add a test case.